controllers/l2serviceattachment_controller.go

```diff
@@ -46,7 +46,7 @@ type L2ServiceAttachmentReconciler struct {
 // +kubebuilder:rbac:groups=eno.k8s.io,resources=l2serviceattachments,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=eno.k8s.io,resources=l2serviceattachments/status,verbs=get;update;patch
 // +kubebuilder:rbac:groups=eno.k8s.io,resources=connectionpoints;l2services,verbs=get;list;watch
-// +kubebuilder:rbac:groups=eno.k8s.io,resources=connectionpoints,verbs=get;list
+// +kubebuilder:rbac:groups=eno.k8s.io,resources=connectionpoints;subnets;routes,verbs=get;list
```

Comment:
I am not sure why we have two entries here ? Why one entry is not good enough ?

Reply:
ok modified it to have 1 entry with line 48.